cogs/common.py

```diff
@@ -123,6 +125,20 @@ async def thanks(self, ctx: Context, member: Member, *, reason: str = None) -> N
             embed.set_image(url="https://media.giphy.com/media/6tHy8UAbv3zgs/giphy.gif")
             await ctx.send(embed=embed)
 
+    @command()
+    async def paste(self, ctx: Context, *, text: str) -> None:
+        """Creates a Paste out of the text specified."""
+        async with self.session.post("https://hasteb.in/documents", data=self._clean_code(text)) as resp:
+            key = (await resp.json())['key']
+            file_paste = 'https://www.hasteb.in/' + key
+
+            await ctx.send(f"FILE PASTE : {file_paste}")
+
+    def _clean_code(self, code: str) -> str:
+        if code.startswith("```") and code.endswith("```"):
+            return "\n".join(code.split("\n")[1:-1])
+        return code.strip("\n")
```

Comment:
This wouldn't work on something like <code>\`\`\`this\`\`\`</code> where the code is on a single line but wrapped in a codeblock (<code>```</code>)

Reply:
i know.

Reply:
so, fix it?